public/static/docs/understanding-dvc/what-is-dvc.md

```diff
@@ -18,15 +18,17 @@ branch or commit.
 
 DVC uses a few core concepts:
 
-- **Experiment**: Equivalent to a Git repository version. Each experiment
-  (extract new features, change model hyperparameters, data cleaning, add a new
-  data source) should be performed in a separate branch and then merged into the
+- **Experiment**: Equivalent to a
+  [Git revision](https://git-scm.com/docs/revisions). Each experiment (extract
```

Comment:
as you are updating this - we can mention that it can be a tag as well (should not be related to merging though)

Reply:
Added "or tag" but left the merging parts, as they seems to be central to this paragraph's idea..?

Reply:
you don't usually merge tags

Reply:
Right, it's a weird sentence now. OK, removed everything after "tag" now. It's obvious what branches are for, anyways.